src/mono/wasm/debugger/BrowserDebugProxy/MonoSDBHelper.cs

```diff
@@ -2427,9 +2402,129 @@ public async Task<JArray> GetObjectValues(int objectId, GetObjectCommandOptions
                         retAfterRemove.Add(item);
                     }
                 }
-                ret = retAfterRemove;
+                objectValues = retAfterRemove;
+            }
+            return objectValues;
+
+            async Task<JArray> GetFieldsValues(List<FieldTypeClass> fields, bool isParent, bool isRootHidden = false)
```

Comment:
`isParent` is not correct. This field is being used to indicate `isOwn`, and gets used as such in the method, IIUC.